sitemap.js

```diff
@@ -1,61 +1,57 @@
-const builder = require('xmlbuilder');
+const jsxString = require('jsx-string');
 
-function createSitemapindex() {
-  const index = builder
-    .create('sitemapindex', { encoding: 'utf-8' })
-    .att('xmlns', 'http://www.sitemaps.org/schemas/sitemap/0.9');
+const XML = content =>
+  `<?xml version="1.0" encoding="utf-8"?>${jsxString(content)}`;
+
+function createSitemapindex(sitemaps = []) {
+  const sitemap = ({ loc, lastmod }) => (
+    <sitemap>
```

Comment:
A template literal might actually work too, but this is more expressive, since you’ll get notified on syntax errors, while in a string you won’t.